src/Symfony/Component/Process/PhpProcess.php

```diff
@@ -27,13 +27,13 @@ class PhpProcess extends Process
     /**
      * Constructor.
      *
-     * @param string $script  The PHP script to run (as a string)
-     * @param string $cwd     The working directory
-     * @param array  $env     The environment variables
-     * @param int    $timeout The timeout in seconds
-     * @param array  $options An array of options for proc_open
+     * @param string      $script  The PHP script to run (as a string)
+     * @param string|null $cwd     The working directory
+     * @param array|null  $env     The environment variables
```

Comment:
It might be worth to add a note what it means to pass `null` and what's the difference to passing an empty array.